herbert_core/admin/@opt_config_manager/private/load_configuration_.m

```diff
@@ -1,48 +1,27 @@
-function conf = load_configuration_(obj,set_config,set_def_only,force_save)
+function obj = load_configuration_(obj,set_config,set_defaults_only,force_save)
 % method loads the previous configuration, which
 % stored as optimal for this computer and, if set_config option is true,
 % configures Horace and Herbert using loaded configurations
 %
-% Returns the structure, containing used configurations info.
+% returns the object, with the configuration data, choosen as default for
+% selected pc type.
+%
 %
-conf = [];
 config_file = fullfile(obj.config_info_folder,obj.config_filename);
 if ~(exist(config_file,'file') == 2)
     warning('No existing configuration file %s found. Current configuration left unchanged',...
         config_file)
     return;
 end
 config_data = xml_read(config_file);
+obj.all_known_configurations_ = config_data;
```

Comment:
is there any need to have this data as both class (`obj.all_known_configurations_`) and local (`config_data`) variable?